en/SetupView/safety_ardupilot.md

```diff
@@ -0,0 +1,202 @@
+
+## Copter
+
+The Copter safety page is shown below. For more information see [Failsafe](ttp://ardupilot.org/copter/docs/failsafe-landing-page.html).
+
+![Safety Setup - Copter (Ardupilot)](../../assets/setup/safety/safety_arducopter.jpg)
+
+
+### Battery Failsafe {#battery_failsafe_copter}
+
+This panel sets the [Battery Failsafe](http://ardupilot.org/copter/docs/failsafe-battery.html) parameters.
+You can set low and critical thresholds for voltage and/or remaining capacity and define the action if the failsafe value is breached. 
+The thresholds can be disabled by setting them to zero.
+
+![Safety Setup - Battery1 Failsafe Triggers (Copter)](../../assets/setup/safety/safety_arducopter_battery1_failsafe_triggers.jpg)
+
+The configuration options are:
+- **Low action** ([BATT_FS_LOW_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-low-act-low-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Critical action** ([BATT_FS_CRT_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-crt-act-critical-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Low voltage threshold** ([BATT_LOW_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-low-volt-low-battery-voltage)) - Battery voltage that triggers the *low action*. 
+- **Critical voltage threshold** ([BATT_CRT_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-crt-volt-critical-battery-voltage))- Battery voltage that triggers the *critical action*. 
+- **Low mAh threshold** ([BATT_LOW_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-low-mah-low-battery-capacity)) - Battery capacity that triggers the *low action*. 
+- **Critical mAh threshold** ([BATT_CRT_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-crt-mah-battery-critical-capacity)) - Battery capacity that triggers the *critical action*. 
+
+
+<!-- Questions
+- What about Battery 2?
+- What actions do BATT_LOW_MAH (capacity) trigger. The ones listed in params don't exist. Is it just same as voltage ones?)
+--> 
+
+### General Failsafe Triggers {#failsafe_triggers_copter}
+
+This panel enables the [GCS Failsafe](http://ardupilot.org/copter/docs/gcs-failsafe.html) 
+and enables/configures the throttle failsafe.
+
+![Safety Setup - General Failsafe Triggers (Copter)](../../assets/setup/safety/safety_arducopter_general_failsafe_triggers.jpg)
+
+The configuration options are:
+- **Ground Station failsafe** - Disabled, Enabled always RTL, Enabled Continue with Mission in Auto Mode, Enabled Always SmartRTL or RTL, Enabled Always SmartRTL or Land.
+- **Throttle failsafe** - Disabled, Always RTL, Continue with Mission in Auto Mode, Always land.
+- **PWM Threshold** ([FS_THR_VALUE](http://ardupilot.org/copter/docs/parameters.html#fs-thr-value-throttle-failsafe-value)) - PWM value below which throttle failsafe triggers.
+
+<!-- Question 
+- no radio failsafe?
```

Comment:
Why no radio link loss failsafe?

Reply:
Hmm, good question. I don't think that was every implemented. What's the parameter for that?

Reply:
I think I was wrong for [radio failsafe](http://ardupilot.org/copter/docs/radio-failsafe.html) - it exists but has no parameter setup (depends on a bunch of other conditions).

There may be some [others missing](http://ardupilot.org/copter/docs/failsafe-landing-page.html)  though: ekf/dcm, polygon fence, crash check, parachute, ADSB.

Reply:
There are lot's of things you can futz with paramters. Only the most important once which are used by most users are exposed in the setup pages ui. The higher end, less used, power user stuff is just left to the parameter editor.

QGC does not support ArduPilot polygon fences or rally points. Long story on that one I'd rather no go into.

Reply:
Makes sense. I will add a note up the top explaining this. You otherwise OK to merge?

Reply:
> You otherwise OK to merge?

yup